Android/Fundamentals/intro-to-android/android-versions.md

```diff
@@ -0,0 +1,49 @@
+# android-versions
+author: tlawson
+
+levels:
+
+  - beginner
+
+type: normal
+
+category: must-know
+
+links:
+
+  - '[Different Versions of Android](https://developer.android.com/about/dashboards/index.html)'
+
+---
+## Content
+
+Each Android has been given a recognizable name which falls in alphabetical order named after some type of dessert. There have been over 12 different versions of Android, Oreo being the latest publicly available one.
+
+Android first started out with Cupcake and Donut, but the first release of Android was Froyo which was version 2.2. Froyo provided huge speed, memory and performance, push notifications and switching between multiple keyboards and languages. Google released version 2.3 which was Gingerbread. This release effort was for user interface and included an implementation to allow field communication, larger screen resolution, and support for multiple cameras. Honeycomb version 3.0 was created which was specific to tablet devices including elements of the new holographics.
+
+Google created Ice Cream Sandwich releasing version 4.0. Ice Cream Sandwich introduce better voice recognition which allow Jelly Bean to bring version 4.1 offering significant performance improvements such as triple buffering and utilizing a fixed frame rate of 60 fps to create a smooth UI.
+
+KitKat Android version 4.4 optimized Android to run on a greater range of devices. A new runtime called ART was introduced as new experimental feature. Android Lollipop version 5.0 introduced a new graphical language called “Material Design” supporting major changes to Android notifications. Android also released Marshmallow version 6.0 and Nougat version 7.0 and of course the latest release is Oreo version 8.0.
+________________________________________
+
+
+---
+## Practice
+
+What is an estimate of Android versions?
```

Comment:
Let's ask users to pick which new device APIs and new major features are available in the newest version, and make sure all of those features (that we choose to list, just pick the most major, relevant to devs) are listed in the insight above